documented size limitation on set script (#28009)
demisto/content: Packs/CommonScripts/Scripts/Set/README.md

## Known Limitations
The script has some limitations with appending to the context in different cases and formats:

> - The *value* argument can't be bigger than 25MB. In general, this is not recommended to ingest large data to context values. 

Reviewer: ```suggestion
- The *value* argument can't be larger than 25MB. In general, it is not recommended to ingest large data to context values. 
```
